src/networkd.c

```diff
@@ -566,6 +566,16 @@ write_network_file(const NetplanNetDefinition* def, const char* rootdir, const c
         }
     }
 
+    /* The ActivationPolicy setting is available in systemd v248+ */
+    if (def->activation_mode) {
+        char* mode;
```

Comment:
We're assigning constant string to this pointer, so I'd suggest to mark it as such.
```suggestion
        const char* mode;
```